src/explore-education-statistics-admin/src/components/PageFooter.tsx

```diff
@@ -19,28 +19,32 @@ const PageFooter = ({ wide }: Props) => {
       >
         <div className="govuk-footer__meta">
           <div className="govuk-footer__meta-item govuk-footer__meta-item--grow">
-            <h2 className="govuk-visually-hidden">Support links</h2>
-            <ul className="govuk-footer__inline-list">
-              {user?.permissions.isBauUser && (
-                <>
+            {user && (
```

Comment:
I've changed this so that the contact us link doesn't show if you're not logged in as it's a protected route (I checked this with Cam)

Reply:
Ahh nice one, ta.